docs/includes/backup-warning.txt

```diff
@@ -0,0 +1,19 @@
+.. note::
+  When dealing with larger backups, the ``securedrop-admin backup`` command may
+  fail with a MemoryError at this stage of the operation: "Fetch the backup
```

Comment:
Nit: Backticks around "MemoryError" would be more consistent with existing style.

Reply:
Should be fixed. :)